sdk/cognitivelanguage/azure-ai-language-conversations/samples/async/sample_analyze_orchestration_app_qna_response_async.py

```diff
@@ -31,42 +32,57 @@ async def sample_analyze_orchestration_app_qna_response_async():
-    conv_endpoint = os.environ["AZURE_CONVERSATIONS_ENDPOINT"]
-    conv_key = os.environ["AZURE_CONVERSATIONS_KEY"]
-    orchestration_project = os.environ["AZURE_CONVERSATIONS_WORKFLOW_PROJECT"]
-
-    # prepare data
-    query = "How do you make sushi rice?",
-    input = ConversationAnalysisOptions(
-        query=query
-    )
+    clu_endpoint = os.environ["AZURE_CLU_ENDPOINT"]
+    clu_key = os.environ["AZURE_CLU_KEY"]
+    project_name = os.environ["AZURE_CLU_ORCHESTRATION_PROJECT_NAME"]
+    deployment_name = os.environ["AZURE_CLU_ORCHESTRATION_DEPLOYMENT_NAME"]
 
     # analyze query
-    client = ConversationAnalysisClient(conv_endpoint, AzureKeyCredential(conv_key))
+    client = ConversationAnalysisClient(clu_endpoint, AzureKeyCredential(clu_key))
     async with client:
-        result = await client.analyze_conversations(
-            input,
-            project_name=orchestration_project,
-            deployment_name='production',
+        query = "How are you?"
+        result = await client.analyze_conversation(
+            task={
+                "kind": "CustomConversation",
+                "analysisInput": {
+                    "conversationItem": {
+                        "participantId": "1",
+                        "id": "1",
+                        "modality": "text",
+                        "language": "en",
+                        "text": query
+                    },
+                    "isLoggingEnabled": False
+                },
+                "parameters": {
+                    "projectName": project_name,
+                    "deploymentName": deployment_name,
+                    "verbose": True
+                }
+            }
         )
 
         # view result
-        print("query: {}".format(result.query))
-        print("project kind: {}\n".format(result.prediction.project_kind))
-
-        print("view top intent:")
-        top_intent = result.prediction.top_intent
-        print("\ttop intent: {}".format(top_intent))
-
-        top_intent_object = result.prediction.intents[0]
-        print("\tconfidence score: {}\n".format(top_intent_object.confidence_score))
+        print("query: {}".format(result.results.query))
+        print("project kind: {}\n".format(result.results.prediction.project_kind))
+
+        # top intent
+        top_intent = result.results.prediction.top_intent
+        print("top intent: {}".format(top_intent))
+        top_intent_object = result.results.prediction.intents[top_intent]
+        print("confidence score: {}".format(top_intent_object.confidence))
+        print("project kind: {}".format(top_intent_object.target_kind))
+
+        if top_intent_object.target_kind == "question_answering":
+            print("\nview qna result:")
+            qna_result = top_intent_object.result
+            for answer in qna_result.answers:
+                print("\nanswer: {}".format(answer.answer))
+                print("answer: {}".format(answer.confidence_score))
```

Comment:
QnA uses `confidence_score` whereas we seem to use `confidence` in other places?
Maybe we should use a swagger override to have this return `confidence` to be consistent....

Reply:
done